backend/src/lib/models/settings.ts

```diff
@@ -24,4 +26,5 @@ export interface SettingsItem {
     date: string;
     time: string;
   };
+  navbarTitle: string;
```

Comment:
Let's make this property optional here (like dateTimeFormat and publishingGuidance) because existing DynamoDB items won't have this property until the next time it gets saved.